bin/rocm-test/scripts/parse_LLNL.sh

```diff
@@ -49,6 +52,10 @@ function parse(){
       fi
       [[ "$line" =~ $testname_regex ]]
       llnltest=${BASH_REMATCH[1]}
+      if [ "$outfile" = "failing-tests.txt" ] && \
```

Comment:
Need the rest of the conditional operator here. I think you are going for `==`.

Reply:
I think string equality with "test" is done with =, not ==. "man test" says:

```
       STRING1 = STRING2
              the strings are equal
```
test is [ of course. This is two conditions, a && b, where a is the [...] part and b is the grep on the following line -- hence the \ continuation.

Reply:
Looks like both `=` and `==` are accepted by bash but `=` is the standard for test. LGTM, thanks.